src/python/pants/backend/python/util_rules/lockfile_metadata.py

```diff
@@ -53,36 +42,46 @@ def from_lockfile(cls, lockfile: bytes) -> LockfileMetadata:
             elif in_metadata_block:
                 metadata_lines.append(line[2:])
 
-        error_suffix = "To resolve this error, you will need to regenerate the lockfile."
+        error_suffix = "To resolve this error, you will need to regenerate the lockfile by running `./pants generate-lockfiles"
```

Comment:
Formatting nit, this is over 100 lines and Black won't attempt to fix. You'd need to use implicit string concatenation.